### ISSUE TITLE
Half-second delayer

### ISSUE THREAD

**Issue by Luke18033** [BEEmod/BEE2-items#1490]
I'm not exactly sure how you would implement this, but how about some way to make half-second delayers? Maybe Start Reversed would subtract half a second from the delay.

**Comment by TeamSpen210** [comment]
What would you need that sort of precision for?

**Comment by Luke18033** [comment]
I wanted to use it for chaining crushers, like Valve did during the beta. I found another way to do it with cubes and lasers, but an actual half (or even better, quarter) second delayer would be much nicer.

**Comment by Konclan** [comment]
What about if crushers had a timer option and output after a certain time after they activate, and infinity will be half a second

**Comment by TheDarkBomber** [comment]
Hammer can use half-second delays as `0.5`.

**Comment by Luke18033** [comment]
@TheDarkBomber - ~~I don't think the Item Configuration Pane is really necessary for this. That's really only for items with lots of properties, or values that need to be set very precisely.~~ Actually, this would be a good idea. Start Enablex would choose between directly choosing timer values, or using configurations from the ICP.

@TeamSpen210 - How about allowing Crushers to be connected together to chain them, making each one go off 0.25 seconds after the last?

**Comment by Luke18033** [comment]
Updated comment 275162535

**Comment by Endermage77** [comment]
Or maybe a delayer item where 1-second is actually 1/2-seconds, and increasing in 1-step increments?

**Comment by libbybapa** [comment]
There are quite a few ways to create smaller delays.  One easy way is to use a laser aimed at a relay, then at a deployed angled panel and one more relay at the other side of the angled panel.  when the laser is activated it takes ~1/2 second for the angled panel to close and the laser to hit the second relay.

**Comment by Luke18033** [comment]
@libbybapa - I prefer not to use "laser logic" like that, it uses a lot more entities than really necessary.

**Comment by libbybapa** [comment]
I agree it is extra objects.  I posted that as a workaround solution, not necessarily as a replacement for the proposed item.

**Comment by Luke18033** [comment]
@TeamSpen210 - The ICP is in the dev version now, add this.
